### ISSUE TITLE
`P_from_to` and `P_to_from` are erroneously zero in DC power flow

### ISSUE THREAD

**Issue by GabrielKS** [NREL-Sienna/PowerFlows.jl#41]
In a power flow output, `P_from_to` is the active power flowing through the line at its `from` bus and `P_to_from` is the active power flowing through the line at its `to` bus. These are not necessarily the same magnitude due to line losses, but they should at least be comparable in magnitude. In the results of an AC power flow, this holds:
<img width="938" alt="AC power flow screenshot (correct behavior)" src="https://github.com/user-attachments/assets/70525978-4c1b-4e31-92fd-4d2af7716e7d">
but in the results of a DC power flow, one of the `P` values for each row seems to be zero:
<img width="929" alt="DC power flow screenshot (incorrect behavior)" src="https://github.com/user-attachments/assets/74b4b47d-fe8f-432f-a827-b40fd69b72f4">

**Comment by rodrigomha** [comment]
I'm confused about this implementation. Here is the code that writes this in the DataFrame: https://github.com/NREL-Sienna/PowerFlows.jl/blob/558a3374d901c984c82e4c9d4d8c15be801f9190/src/post_processing.jl#L491

```
# get flows on each line
    P_from_to_vect = zeros(length(branches))
    Q_from_to_vect = zeros(length(branches))
    P_to_from_vect = zeros(length(branches))
    Q_to_from_vect = zeros(length(branches))
    for i in 1:length(branches)
        if branch_flow_values[i] >= 0
            P_from_to_vect[i] = branch_flow_values[i]
            P_to_from_vect[i] = 0
        else
            P_from_to_vect[i] = 0
            P_to_from_vect[i] = branch_flow_values[i]
        end
    end
```

Based on this implementation, if the solution is positive for the branch, it assigns the proper value to the from_to (probably correct). However if the solution is negative, it's assigned the same value to the `to_from` (probably incorrect). I'm guessing that in the solution of the DC PowerFlow, the solution is always for `from_to`, so in this case what it needs to happen is the following:
```
# get flows on each line
    P_from_to_vect = zeros(length(branches))
    Q_from_to_vect = zeros(length(branches))
    P_to_from_vect = zeros(length(branches))
    Q_to_from_vect = zeros(length(branches))
    for i in 1:length(branches)
        if branch_flow_values[i] >= 0
            P_from_to_vect[i] = branch_flow_values[i]
            P_to_from_vect[i] = 0
        else
            P_from_to_vect[i] = 0
            P_to_from_vect[i] = -branch_flow_values[i]
        end
    end
```
so the flows are always positive.

**Comment by jd-lara** [comment]
Yeah, it seems this the post processing code isn't implemented correctly.

**Comment by jd-lara** [comment]
Let's resolve it to have the P_from_to = -P_to_from and report it like that.

**Comment by rodrigomha** [comment]
Closed via #45
